OSS-Fuzz issue 48673

OSS-Fuzz has found a bug in this project. Please see https://oss-fuzz.com/testcase?key=5355130886684672 for details and reproducers.

This issue is mirrored from https://bugs.chromium.org/p/oss-fuzz/issues/detail?id=48673 and will auto-close if the status changes there.

If you have trouble accessing this report, please file an issue at https://github.com/google/oss-fuzz/issues/new.

```
--- a/pkg/report/report_test.go
+++ b/pkg/report/report_test.go
@@ -415,6 +415,8 @@ func TestFuzz(t *testing.T) {
                "BUG: login: [0.0] ",
                "cleaned vnode",
                "kernel:",
+               `BUG:
+Code: `,
        } {
                Fuzz([]byte(data)[:len(data):len(data)])
        }
```

gives me:
```
$ go test -vet=off -run=Fuzz ./pkg/report
--- FAIL: TestFuzz (0.00s)
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0xe8 pc=0x605622]

goroutine 10 [running]:
testing.tRunner.func1.2({0x64bc40, 0xe18cd0})
	testing/testing.go:1209 +0x24e
testing.tRunner.func1()
	testing/testing.go:1212 +0x218
panic({0x64bc40, 0xe18cd0})
	runtime/panic.go:1038 +0x215
github.com/google/syzkaller/pkg/report.(*linux).parseOpcodes(0xc0001202c0, {0x0, 0xc0001f5ab0})
	pkg/report/linux.go:561 +0x62
github.com/google/syzkaller/pkg/report.(*linux).processOpcodes(0xc0003b59a0, {0x0, 0x0})
	pkg/report/linux.go:477 +0x38
github.com/google/syzkaller/pkg/report.(*linux).decompileReportOpcodes(0xb, {0xc000017400, 0xc, 0x10})
	pkg/report/linux.go:645 +0x1da
github.com/google/syzkaller/pkg/report.(*linux).Symbolize(0xc0001202c0, 0xc00020a7e0)
	pkg/report/linux.go:370 +0x55
github.com/google/syzkaller/pkg/report.(*Reporter).Symbolize(0xc0003bacd0, 0xc00020a7e0)
	pkg/report/report.go:229 +0x44
github.com/google/syzkaller/pkg/report.Fuzz({0xc0000173a0, 0xb, 0xb})
	pkg/report/fuzz.go:26 +0x17b
github.com/google/syzkaller/pkg/report.TestFuzz(0x9583c0)
	pkg/report/report_test.go:421 +0x172
testing.tRunner(0xc000082ea0, 0x844850)
	testing/testing.go:1259 +0x102
created by testing.(*T).Run
	testing/testing.go:1306 +0x35a
FAIL	github.com/google/syzkaller/pkg/report	0.175s
FAIL
```

--- a/pkg/report/fuzz.go
+++ b/pkg/report/fuzz.go
@@ -70,8 +70,13 @@ var fuzzReporters = func() map[string]*Reporter {
 		if os == targets.Windows {
 			continue
 		}
+		target := targets.Get(os, targets.AMD64)
+		if target == nil {
+			continue
+		}
 		cfg := &mgrconfig.Config{
 			Derived: mgrconfig.Derived{
+				SysTarget:  target,
 				TargetOS:   os,
 				TargetArch: targets.AMD64,
 			},

